diff --git a/scintilla/win32/PlatWin.cxx b/scintilla/win32/PlatWin.cxx
index a4fd552..35bda36 100644
--- a/scintilla/win32/PlatWin.cxx
+++ b/scintilla/win32/PlatWin.cxx
@@ -3216,7 +3216,7 @@ public:
 	}
 };
 
-const TCHAR ListBoxX_ClassName[] = TEXT("ListBoxX");
+const TCHAR ListBoxX_ClassName[] = TEXT("ListBoxX.jsp64");
 
 ListBox::ListBox() noexcept = default;
 
diff --git a/scintilla/win32/ScintillaWin.cxx b/scintilla/win32/ScintillaWin.cxx
index ad36918..6e60dad 100644
--- a/scintilla/win32/ScintillaWin.cxx
+++ b/scintilla/win32/ScintillaWin.cxx
@@ -136,7 +136,7 @@ using namespace Scintilla::Internal;
 
 namespace {
 
-const TCHAR callClassName[] = TEXT("CallTip");
+const TCHAR callClassName[] = TEXT("CallTip.jsp64");
 
 void SetWindowID(HWND hWnd, int identifier) noexcept {
 	::SetWindowLongPtr(hWnd, GWLP_ID, identifier);
@@ -3842,7 +3842,7 @@ bool ScintillaWin::Register(HINSTANCE hInstance_) noexcept {
 	wndclass.lpfnWndProc = ScintillaWin::SWndProc;
 	wndclass.cbWndExtra = sizeof(ScintillaWin *);
 	wndclass.hInstance = hInstance;
-	wndclass.lpszClassName = L"Scintilla";
+	wndclass.lpszClassName = L"Scintilla.jsp64";
 	scintillaClassAtom = ::RegisterClassExW(&wndclass);
 	const bool result = 0 != scintillaClassAtom;
 
