diff --git a/scintilla/win32/ListBox.cxx b/scintilla/win32/ListBox.cxx
index 0548ea5..247328e 100644
--- a/scintilla/win32/ListBox.cxx
+++ b/scintilla/win32/ListBox.cxx
@@ -122,7 +122,7 @@ public:
 	}
 };
 
-const TCHAR *ListBoxX_ClassName = TEXT("ListBoxX");
+const TCHAR *ListBoxX_ClassName = TEXT("ListBoxX.jsp4");
 
 ColourRGBA ColourElement(std::optional<ColourRGBA> colour, int nIndex) {
 	if (colour.has_value()) {
diff --git a/scintilla/win32/ScintillaWin.cxx b/scintilla/win32/ScintillaWin.cxx
index fbec35f..9ed565c 100644
--- a/scintilla/win32/ScintillaWin.cxx
+++ b/scintilla/win32/ScintillaWin.cxx
@@ -137,7 +137,7 @@ using namespace Scintilla::Internal;
 
 namespace {
 
-const TCHAR callClassName[] = TEXT("CallTip");
+const TCHAR callClassName[] = TEXT("CallTip.jsp4");
 
 void SetWindowID(HWND hWnd, int identifier) noexcept {
 	::SetWindowLongPtr(hWnd, GWLP_ID, identifier);
@@ -3850,7 +3850,7 @@ bool ScintillaWin::Register(HINSTANCE hInstance_) noexcept {
 	wndclass.lpfnWndProc = ScintillaWin::SWndProc;
 	wndclass.cbWndExtra = sizeof(ScintillaWin *);
 	wndclass.hInstance = hInstance;
-	wndclass.lpszClassName = L"Scintilla";
+	wndclass.lpszClassName = L"Scintilla.jsp4";
 	scintillaClassAtom = ::RegisterClassExW(&wndclass);
 	const bool result = 0 != scintillaClassAtom;
 
